context/ip_allocator.go

```diff
@@ -95,13 +91,12 @@ func (a *IPAllocator) Allocate(imsi string) (net.IP, error) {
 		return nil, errors.New("ip allocation failed" + err.Error())
 	} else {
 		// smfCount := MongoDBLibrary.GetSmfCountFromDb()
-
-		smfCountStr := os.Getenv("SMF_COUNT")
-		smfCount, _ := strconv.Atoi(smfCountStr)
-		ip := IPAddrWithOffset(a.ipNetwork.IP, int(offset)+(int(smfCount)-1)*5000)
```

Comment:
@UpadhyayaAnkur 
Could you please help me to understand offset calculation with multiple SMF's here ?
`ip := IPAddrWithOffset(a.ipNetwork.IP, int(offset)+(int(smfCount)-1)*5000)`

Reply:
I think calculation was introduced when the capability to deploy multiple SMFs was added. What the formula does is to assume that each SMF will handle/assign up to 5000 UEs (IPs). So, it assigns "consecutive" IP addresses to UEs. However, there are two problems with this way of assigning IP addresses: 1) the environment variable "SMF_COUNT" does not exist and the IP addresses that are assigned start with a 5000 "shift"; and 2) this formula ignores the "mask" (i.e., if a user only wants a 20-bit mask or higher)

Reply:
@gatici I shall update logic details sometime today

Reply:
@gatici This way of dividing IP-Addresses range across multiple SMF instances doesn't look clean(shift of 5000 address across each SMF instance). 
Probably, better approach could have been to reserve few host address bits for storing SMF instance(sub-block) and remaining for host id. This way each SMF instance can enjoy entire sub-block addresses. Please free to change existing logic